.github/workflows/test.yml

```diff
@@ -45,6 +45,24 @@ jobs:
         with:
           command: test
 
+  test:
```

Comment:
Is it because there are two tags called "test"?  
![image](https://user-images.githubusercontent.com/30254428/166233974-dc7e32f2-f400-41f6-87f5-9e126a69930e.png)

https://github.com/dada-lang/dada/actions/runs/2257562812/workflow